frontend/src/views/Admin.vue

```diff
@@ -0,0 +1,11 @@
+<template>
```

Comment:
Also for structure wise, for all of the views, create a folder called modules and in the modules folder put the different views within them,

modules/admin/Admin.vue

I shouldve added this structure when I first made the repo, that is on me. Modules are organized by feature this way (admin), views should have the global pages of the project like Home, PageNotFound, OAuth call back, etc